### PR TITLE
[32863]Add a link into news widget to create the first news when there is currently no news

### PR DESCRIPTION
**Pull request by oliverguenther**
- [ ] Requires API change to allow showing the URL

https://community.openproject.com/projects/openproject/work_packages/32863/activity

### CODE REVIEW

File: frontend/src/app/modules/common/no-results/no-results.component.ts [hunk 1]
```diff
@@ -26,7 +26,8 @@
 // See docs/COPYRIGHT.rdoc for more details.
 //++
 
-import {Component, Input, HostBinding} from '@angular/core';
+import {Component, HostBinding, Input} from '@angular/core';
+
```
Comment (openprojectci) [inline frontend/src/app/modules/common/no-results/no-results.component.ts:31]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/c30cd351cbaae9715a5df0fcb89c701d4b71b4d0/frontend/src/app/modules/common/no-results/no-results.component.ts#L31"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>

**Comment by lindenthal** [comment]
Hi @bsatarnejad,
can you please add the respective work package to this pull request?

I just had another look on the dev branch. Currently there is no create button in the news widget?

image

@oliverguenther Is this PR a good option to also address this?

Best
Niels

**Comment by bsatarnejad** [comment]
Hi @lindenthal 
The respective work package is 32863.
We should make some changes to the API to finish the last part of this work package which is related to users right to manage the news. I think because of this it is not merged now.
When there is not any news, a link will be shown that moves the user forward to create the first news.

**Comment by oliverguenther** [comment]
Closing for now to clean up open PRs. It doesn't look like we're prioritizing this anytime soon

**Pull request closed without merging**